src/common/low_precision_transformations/src/layer_transformation.cpp

```diff
@@ -4,7 +4,7 @@
 
 #include <low_precision/layer_transformation.hpp>
 #include <low_precision/network_helper.hpp>
-
+#include "core/src/itt.hpp"
```

Comment:
remove?

Reply:
> remove?

@ilya-lavrenov Add this header is because this macro `OV_PASS_CALLBACK(m);` (used in line 456) define in it. I also think this change is ugly, any other suggestions?

Reply:
I think we can move this callback to: https://github.com/openvinotoolkit/openvino/blob/master/src/common/conditional_compilation/include/openvino/cc/pass/itt.hpp